NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+<packageSources>
+ <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+ <add key="Roslyn nightly builds" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
we should not be using the nightly build feed. I'd rather internalize the DLLs so that we can publish a nuget package